README.md

```diff
@@ -297,143 +213,121 @@ More on this soon.
 -   `vendor` holds non-gem software that our app will use. Don't worry too much
     about this one for now.
 
-For now, you only need to think about `app`, `config`, and `db` - you probably
-won't be touching any code outside of those three directories. How is that
-possible? Because Rails actually builds out most of these files and folders for
-you, every time you use it to create a new application. That's why it's called
-a 'framework' - it gives you the skeleton for a brand new app, which you can
-then customize.
-
 Let's dive into the `app` directory. This app in particular has more going on
 than yours probably will, but it still has all the basic components.
 
 ```bash
 ./app
-├── assets
-│   ├── images
-│   └── stylesheets
-│       └── application.css
+├── channels
 ├── controllers
-│   ├── application_controller.rb
-│   ├── concerns
-│   ├── games_controller.rb
-│   ├── open_read_controller.rb
-│   ├── protected_controller.rb
-│   └── users_controller.rb
-├── helpers
-│   └── application_helper.rb
+├── jobs
 ├── mailers
 ├── models
-│   ├── concerns
-│   │   ├── authentication.rb
-│   │   ├── listen_notify.rb
-│   │   └── tic_tac_toe_validation.rb
-│   ├── game.rb
-│   └── user.rb
 ├── serializers
-│   ├── game_serializer.rb
-│   └── user_serializer.rb
 └── views
-    └── layouts
-        └── application.html.erb
 ```
 
 Three of these directories should jump out at you: `controllers`, `models`, and
 `views`. Each holds the different Ruby files that Rails uses to handle the
 respective responsibilities of MVC.
 
-Don't worry about `assets`, `serializers`, `mailers`, or `helpers` for now.
+Don't worry about `assets`, `serializers`, `mailers`, or `helpers` for now. In
+API-style applications, `serializers` replace all the views that we care about
+for this program, so we can safely skip the `views` folder as well.
 
 ## Demo: Building our first Rails App
 
 Now that we've conceptually wrapped our heads around what goes into a backend
 lets make one ourselves.
 
-**This is intended to show you how powerful Rails is; not so you understand!!**
-**We will go further into _how_ Rails works later on**
+**WARNING:** Do not ask "why?". None of this should "make sense", yet. Instead,
```

Comment:
Maybe don't ask why **right now**? I worry this kind of language might inhibit them from asking in later lessons when it's important.